### ISSUE TITLE
Review Updated SRS

### ISSUE THREAD

**Issue by alaapgrandhi** [alaapgrandhi/equivariant-sensor-fusion#16]
I updated the SRS document and wanted to add a list of changes that I made here to make it easier to evaluate.

Specifically, I made the following changes:
- Changed the variables to be wrapped in dollar signs
- Added missing symbol table entries
- Removed unused Solar Water Heater project symbol table entries
- Fixed formatting for true positives (TP), false positives (FP), and false negatives (FN) in the symbol table
- Clarified requirements scoping to avoid a conflict with the introduction
- Gradient Descent was added to the ADAM optimizer constraint (since ADAM is a gradient-descent algorithm)
- Rationale was linked to where relevant
- GS2 updated to include the trained model from GS1
- Added links from assumptions to where they are called
- P_c^{*} is left as is in TM:Intrin as it is only defined and used there (I added a triangle equals sign to make this clearer)
- Added X_{c}, Y_{c}, and Z_{c} to the symbol table
- Moved placement of certain variables around in theoretical models to enhance clarity
- Changed "set of controllable parameters" to "vector of controllable parameters" and specified that they are all in $R$
- Made mAP formatting more consistent
